.github/workflows/downsize-insurance.yml

```diff
@@ -0,0 +1,19 @@
+name: Downsize Fly.io VM
```

Comment:
The intent of this file is as a failsafe against runaway Fly.io bills, just in case sync.yml fails to run. 
FWIW using `always()` in the scale down step does seem to be working in testing so far.